.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,14 +4,22 @@ about: Report a bug encountered while operating KKP
 labels: kind/bug
 ---
 
-### What happened?
+### What happened
 
-### What should happen?
+### Expected behavior
 
-### How to reproduce?
+### How to reproduce
 
 ### Environment
 
 - KKP Version:
 - Shared or separate master/seed clusters?:
 - Master/seed cluster provider:
+
+### Current workaround
+
+### Affected user persona
+
+### Business goal to be improved
+
+### Metric to be improved
```

Comment:
Could we maybe explain those a bit via comments (`<-- Comment here -->` should work in templates I think)? It's not obvious to me what should be added here and we should keep in mind that people outside of Kubermatic GmbH will also get those templates when opening tickets with us :-)

Reply:
I do not think those questions are suitable for a public, open source bugtracker. This makes it sounnd like "Oh, you're not a Kubermatic customer? Well f you then!".

Please do not mix business development strategies with a simple bugtracker. This will drive everyone away from this project.